00-HTML-CSS-basics/HTML&CSS/2.1style.css

```diff
@@ -0,0 +1,36 @@
+body {font-size: 14px;}
+
+.div {width: 960px;
```

Comment:
A good practice is to order the rules by alphabetical order, and also try to format the rules' block like this:
selector {
  rule1: value1;
  rule2: value2;
  ...
}